wdl2cwl/main.py

```diff
@@ -284,10 +286,7 @@ def get_expr_string(self, wdl_expr_string: WDL.Expr.String) -> str:
         string = ""
         parts = wdl_expr_string.parts
         for index, part in enumerate(parts[1:-1], start=1):
-            if isinstance(
-                part,
-                (WDL.Expr.Placeholder, WDL.Expr.Apply, WDL.Expr.Get, WDL.Expr.Ident),
-            ):
+            if isinstance(part, WDL.Expr.Placeholder):
```

Comment:
only placeholders in strings have been observed. No other expression seems possible to be embedded in a string.